### PR TITLE
Adds ERP 

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request
Ports much of the arousal module from Citadel. Adds genitals, cum, climaxing and all sorts of fun stuff.

## Why It's Good For The Game
Now you can treat catgirls the way they truly deserve. BDSM module follow-up with breath, electro and knife play coming soon. 

## Changelog
:cl:
add: Penis, testicles, breasts, vagina 
add: Flavor text 
add: Traits for nymphomania, masochism, perma and never arousal 
add: Face and ass slapping 
add: Cum 
add: Partner climaxing
add: Dildos
add: Makes April 1st the anniversary day for ERP
/:cl:

### CODE REVIEW

**Comment by Isy232** [comment]
Shoulda just done something small, like replacing the handcuff sprites with fuzzy ones.

**Comment by WhyIsCaeciliusTaken** [comment]
why

**Comment by Cenrus** [comment]
Mother of God. Don't read the file diffs

**Comment by Arrfur** [comment]
### 👍 
### good change! this is well needed for the bee station community!





_PS: read the date @Cenrus @WhyIsCaeciliusTaken @Isy232_

**Comment by derpymcfuckwit** [comment]
im about to cum

**Comment by ghost** [comment]
sissy hypnosis? nah fuck that, we doin monkey hypnosis

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -804,3 +804,85 @@
...
+	if(prob(20))
+		M.emote("nya")
+	if(prob(20))
+		to_chat(M, "<span class = 'notice'>[pick("Headpats feel nice.", "Backrubs would be nice.", "Mew")]</span>")
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/list/adjusted = H.adjust_arousal(5,aphro = TRUE)
+		for(var/g in adjusted)
+			var/obj/item/organ/genital/G = g
+			to_chat(M, "<span class='userlove'>You feel like playing with your [G.name]!</span>")
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:861]: If you have multiple genitals would this not loop through all of them and spam the chat a little?

**Review by PowerfulBacon — COMMENTED** [review]
very good code, well written. thanks for porting

**Comment by BettonnCZ** [comment]
imagine adding this

**Comment by BettonnCZ** [comment]
add it

**Comment by Blueturbo47** [comment]
reopen this honk

**Comment by ItsTisme** [comment]
WHAT DO YOU MEAN WE DIDNT SPEED MERGETHIS?!?!?!?!

**Pull request closed without merging**